locator/tablets.hh

```diff
@@ -399,6 +399,9 @@ struct load_stats {
     // Capacity in bytes for data file storage.
     std::unordered_map<host_id, uint64_t> capacity;
 
+    // Critical disk utilization check for each host.
+    std::optional<std::unordered_map<locator::host_id, bool>> critical_disk_utilization;
```

Comment:
Why does it need to be optional?
Is it any different from an empty map when initialized?